packages/theme/stories/characteristics/characteristics.stories.mdx

```diff
@@ -53,6 +57,101 @@ For example, a Button is a component that a user can perform an action on, so we
 Characteristics are both light/dark and density scoped, meaning that the value the characteristic points to will change according to the values given in the nearest ToolkitProvider. Considering the above, we would get that in light
 theme, --uitk-actionable-primary-background resolves to `var(--uitk-action-primary-background)`, which in light theme is `var(--uitk-color-grey-60)` and in dark theme is `var(--uitk-color-grey-300)`, of which foundation tokens then in turn resolve to their respective RGBA values.
 
+## Emphasis
+
+### Using emphasis
+
+Some characteristic tokens have a concept of emphasis. Components can apply characteristics with emphasis to make them stand out on the page amongst the rest of the UI, or have less prominence so that other components appear exaggerated.
+
+An example of a characteristic token which has emphasis levels is `--uitk-editable-background-` which has the three levels defined, `low`, `medium`, and `high`. By default, the FormField component has been styled to use the `--uitk-editable-background-medium` token in a local variable:
+
+```css
+.uitkFormField {
+  --form-field-background: var(--uitk-editable-background-medium);
+}
+```
+
+Which, in the UITK theme, just makes the background white. Let's make it appear on a purple Panel for demo purposes:
+
+<Panel style={{ background: "var(--uitk-color-purple-10)" }}>
+  <FormField
+    style={{ width: "242px" }}
+    label="High emphasis form field"
+    helperText="Helper text value"
+  >
+    <Input defaultValue="Value" />
+  </FormField>
+</Panel>
+
+Perhaps though, you would like a FormField to have more emphasis on the page. In this case, you can pass in the `uitkEmphasisHigh` className directly to FormField. This will then override the above value:
+
+```css
+.uitkEmphasisHigh.uitkFormField {
+  --form-field-background: var(--uitk-editable-background-high);
+}
+```
+
+Giving the following appearance:
+
+<Panel style={{ background: "var(--uitk-color-purple-10)" }}>
+  <FormField
+    style={{ width: "242px" }}
+    label="High emphasis form field"
+    helperText="Helper text value"
+    className="uitkEmphasisHigh"
+  >
+    <Input defaultValue="Value" />
+  </FormField>
+</Panel>
+
+Or otherwise, maybe we want it to have a lower emphasis... just as easy: pass in the `uitkEmphasisLow` className to FormField, and voila - a transparent background:
+
+<Panel style={{ background: "var(--uitk-color-purple-10)" }}>
+  <FormField
+    style={{ width: "242px" }}
+    label="High emphasis form field"
+    helperText="Helper text value"
+    className="uitkEmphasisLow"
+  >
+    <Input defaultValue="Value" />
+  </FormField>
+</Panel>
+
+### Extending emphasis to more components
+
+It's easy to add emphasis to components that the UITK doesn't offer by default. Suppose you want to add low emphasis to a Card. Currently, Card makes use of the `---uitk-container-background-medium` token for all of it's styling purposes. Here's how it looks by default:
```

Comment:
"It's easy to add emphasis to components that the UITK doesn't offer by default."  reads a bit oddly to me. 

How about

"Not all UITK components support emphasis by default. This capability can easily be added though."

Reply:
done